artifactor/plugins/reporter.py

```diff
@@ -363,9 +363,11 @@ def skip_test(self, test_location, test_name, skip_data):
         return None, {'artifacts': {test_ident: {'skipped': skip_data}}}
 
     @ArtifactorBasePlugin.check_configured
-    def start_test(self, test_location, test_name, slaveid):
+    def start_test(self, test_location, test_name, slaveid, tier=None):
         test_ident = "{}/{}".format(test_location, test_name)
-        return None, {'artifacts': {test_ident: {'start_time': time.time(), 'slaveid': slaveid}}}
+        return None, {'artifacts': {test_ident: {
+            'start_time': time.time(), 'slaveid': slaveid, 'tier': tier or "N/A"}}
```

Comment:
I assume it's ok that `tier` is a number when set but a string (`'N/A'`) when not set?